### ISSUE TITLE
Both sides sync

### ISSUE THREAD

**Issue by XargonWan** [x0b/rcx#71]
#### Pre-Submission checklist

 - [x] There is no existing issue that already asks for this feature.<br />
       _If no, please add a_ +1, _and maybe a comment if you have more to add._
 - [x] The feature already exists in rclone (on your PC or in Termux)
 - [ ] I am prepared to help make this feature

#### What version of RCX are you using (About -> App version)?
1.11.4

#### What problem are you trying to solve?
TLDR: Sync a folder on multiple devices.

I want to sync my RetroArch saves between multiple devices.

#### What should RCX be able to do differently with help this problem?
I wish to the folders on both ways, but when I choose sync I can only decide to sync the local copy on the cloud or the cloud copy on local, I wish to have an exact carbon copy of that folder on both the cloud (Dropbox in this case) and locally.

**Comment by x0b** [comment]
Thank you for your enhancement idea. Yet I'm going to close this issue.

XargonWan wrote:
>  - [x] The feature already exists in rclone (on your PC or in Termux)

According to ncw himself [[1]](https://github.com/rclone/rclone/issues/4213#issuecomment-626353074), [[2]](https://forum.rclone.org/t/semi-rclone-bisync-two-way-sync-with-rclone-copy/8995/2):

> rclone doesn’t have a bisync solution at the moment.
> ...
> It isn't a trivial feature to implement.

It sounds like you need a file synchronisation tool, but rclone - on which RCX is built - is not. This problem is so complicated that [it took Dropbox years and multiple attempts to "solve"](https://dropbox.tech/infrastructure/rewriting-the-heart-of-our-sync-engine).

**Comment by cjnaz** [comment]
As a stopgap, consider [https://github.com/cjnaz/rclonesync-V2]()
